### PR TITLE
Create example snap

### PR DESCRIPTION
**Pull request by jpm-canonical**
Add label image python script
Use python3.8 from deadsnakes ppa on core24 base

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,85 @@
-# tf-label-image-snap
+# TensorFlow Lite label image snap
+
+This example bundles an example TensorFlow Lite script with all the required dependencies.
```
Comment (farshidtz) [inline README.md]:
```suggestion
This snap bundles an example TensorFlow Lite script with all the required dependencies.
```

File: README.md [hunk 2]
```diff
@@ -1 +1,85 @@
...
+The base of this snap is core24, which is based on the current latest LTS release of Ubuntu, namely Ubuntu 24.04 Noble Numbat, and should provide the longest support.
```
Comment (farshidtz) [inline README.md]: Changing to make this not relative to the current date.
```suggestion
The base of this snap is core24, which is based on Ubuntu 24.04 LTS, with up to 12 years of support.
```

File: README.md [hunk 3]
```diff
@@ -1 +1,85 @@
...
+INFO: Created TensorFlow Lite XNNPACK delegate for CPU.
+0.919721: 653:military uniform
+0.017762: 907:Windsor tie
+0.007507: 668:mortarboard
+0.005419: 466:bulletproof vest
+0.003828: 458:bow tie, bow-tie, bowtie
+time: 28.502ms
+```
+
+Any other image can be specified using the `-i` argument. The snap has read access to the user's home directory, so any path under $HOME (or `~`) can be specified. Example:
```
Comment (farshidtz) [inline README.md]: Keep in mind that the home interface only auto connects on classic systems.

File: label_image_lite.py [hunk 4]
```diff
@@ -0,0 +1,130 @@
+# Copyright 2018 The TensorFlow Authors. All Rights Reserved.
```
Comment (farshidtz) [inline label_image_lite.py]:
```suggestion
# Copyright 2018 The TensorFlow Authors.
# Copyright 2024 Canonical Ltd.
```

File: README.md (hunk 3 above)
Comment (jpm-canonical) [reply README.md]: Ah ok. Let's keep this in mind for now. I'll likely run into it when testing on core, at which time the readme will be updated with the correct steps to use it on core.

**Pull request merged**